Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY requirements.txt /usr/src/app/
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . /usr/src/app
```

Comment:
Dockerfile should define an entrypoint.  Currently it just inherits the python interpreter entrypoint from the base image.  Should be `python server.py` or similar.

Reply:
I'm working on the new Dockerfile which will launch both vue and python servers in the single container for us. This Dockerfile has to be scraped. Will push once done.